### ISSUE TITLE
11/09/23_NYU_Android_talkback_PurpleRide_User cannot submit the feedback in ride completed screen _1980

### ISSUE THREAD

**Issue by krupal-14** [nammayatri/nammayatri#3101]
#1980 
[User cannot write feed back in ride completed screen](https://drive.google.com/file/d/1MbS-pRIf3SJ-_DwipTLcwCB5pCnYNC3C/view?usp=sharing)

Steps To Reproduce
1.Launch the apk.
2.Login with valid credentials.
3.Complete a ride.
4.Click on feedback and observe

Expected Result
User should able to submit the feedback.

Actual result
User is not able submit feedback

**Comment by MercyQueen** [comment]
Fixed.
